data/boarding missions.txt

```diff
@@ -24,6 +24,343 @@ mission "Assisting Merchant"
 	# What the destination is doesn't matter, because you always 'decline' this.
 	destination Earth
 
+mission "Assisting Merchant (Hello)"
+	assisting
+	repeat
+	to offer
+		random < 10
+	source
+		government Merchant
+	on offer
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "Hello there fellow captain!"`
+				decline
+	destination Earth
+
+mission "Assisting Merchant (Blessing)"
+	assisting
+	repeat
+	to offer
+		random < 10
+	source
+		government Merchant
+	on offer
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "May fair winds and bright skies go with you!"`
+				decline
+	destination Earth
+
+mission "Assisting Merchant (Force)"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government Merchant
+	on offer
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "May the Force be with you!"`
+				decline
+	destination Earth
+	
+mission "Assisting Merchant (Navy)"
+	assisting
+	repeat
+	to offer
+		random < 10
+	source
+		government Merchant
+	on offer
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "The best way to be safe from pirates is to stick to systems patrolled by the Navy."`
```

Comment:
These messages that the ship captains are giving you seem a bit cheesy, again going back to when I said the mission variants seem pointless. If you're looking to increase the variety in what can be said to you when you assist merchants, then just use branches with the random condition instead of making multiple missions.